.github/workflows/release.yaml

```diff
@@ -256,6 +256,42 @@ jobs:
           signing-key: ${{ secrets.COSIGN_PRIVATE_KEY }}
           signing-password: ${{ secrets.COSIGN_PASSWORD }}
 
+  build-gcr-deployer:
+    name: Build GCR deployer image
+    environment: Release
+    runs-on: ubuntu-latest
+    needs: [ prepare ]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      - name: Login to GCR
+        uses: redhat-actions/podman-login@4934294ad0449894bcd1e9f191899d7292469603 # v1.7
+        with:
+          registry: gcr.io
+          username: ${{ secrets.GCR_USERNAME }}
+          password: ${{ secrets.GCR_JSON_KEY }}
+      - name: Build target
+        id: build-image
+        uses: redhat-actions/buildah-build@7a95fa7ee0f02d552a32753e7414641a04307056 # v2.13
+        with:
+          oci: true
+          context: ./config/helm/
+          containerfiles: |
+            ./config/helm/Dockerfile
+          tags: ${{ needs.prepare.outputs.version }}
+          image: gcr.io/${{ secrets.GCR_REPOSITORY_DEPLOYER }}
+          extra-args: |
+            --annotation=com.googleapis.cloudmarketplace.product.service.name=services/dynatrace-operator-dynatrace-marketplace-prod.cloudpartnerservices.goog
+      - name: Push To gcr
+        id: push-to-gcr
+        uses: redhat-actions/push-to-registry@5ed88d269cf581ea9ef6dd6806d01562096bee9c # v2.8
+        with:
+          image: ${{ secrets.GCR_REPOSITORY_DEPLOYER }}
+          registry: gcr.io
+          tags: ${{ steps.build-image.outputs.tags }}
+          username: ${{ secrets.GCR_USERNAME }}
```

Comment:
We wouldn't need a secret for this. Would make it clear that it is `_json_key` aka. service account key.

Reply:
And not oauth, which expires quickly

Reply:
@gkrenn i can't change it for now because we need to change it for another GCR related image push/pull/login steps